test/distributed/fsdp/test_fsdp_meta.py

```diff
@@ -0,0 +1,315 @@
+# (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
```

Comment:
I have never actually known what the protocol was for headers like this :laughing: